src/sui/themes/rtd-site/globals/site.overrides

```diff
@@ -52,6 +52,75 @@ body {
   color: @black;
 }
 
+/*-------------------
+Responsive alignment
+--------------------*/
```

Comment:
This would be a great place to document what normal usage of these classes looks like. Just a brief example would help those not familiar with normal alignment rules/etc.